### PR TITLE
Add recipe for polish bigos

### PR DESCRIPTION
**Pull request by kKondratek**

### CODE REVIEW

**Comment by printplaatreparatie** [comment]
I love this recipe! I'm not from Poland, but for me are tomato paste and garlic essential for the good taste. @kKondratek what do you think?

File: src/polish-bigos.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+- Dried mushrooms (optional): 30 g
+- Dry red wine: a little (to pour)
+- Allspice
+- Bay leaf
+
+## Directions
+
+1. Dice pork and cut onion into small pieces.
+2. Fry onion in oil in a large pot (the whole dish must fit there), then add pork and fry it too.
+3. Pour one and half glasses of warm water with salt and pepper (not to much salt!).
```
Comment (hallzy) [inline src/polish-bigos.md:27]:
```suggestion
3. Pour one and half glasses of warm water with salt and pepper (not too much salt!).
```

**Review by hallzy — CHANGES_REQUESTED** [review]

**Comment by kKondratek** [comment]
> I love this recipe! I'm not from Poland, but for me are tomato paste and garlic essential for the good taste. @kKondratek what do you think?

I heard about this variation :) never tried before but I'm sure it's great too

**Review by DGKSK8LIFE — APPROVED** [review]

**Comment by m-chrzan** [comment]
I would simplify the name to just "bigos".

**Comment by kKondratek** [comment]
> I would simplify the name to just "bigos".
@m-chrzan I agree, just renamed it

**Review by hallzy — APPROVED** [review]

**Pull request closed without merging**